generator.rb

```diff
@@ -0,0 +1,57 @@
+def is_vowel?(letter)
+  if "aeiou".include?(letter)
+    return true
+  else
+    return false
+  end
+end
+
+# Make a method named run_generator. This method has no parameters. Start this method by outputting a prompt to ask for a first input word A, and accept some input. Then, ask for a second input word B, and accept some input.
+def run_generator
+  # Get words to combine from user
+  print "What is the first word to combine? "
+  word_a = gets.chomp.downcase
+  until word_a.length >= 2
+    puts "Word must be greater than 1 character! Try again: "
+    word_a = gets.chomp.downcase
+  end
+
+  # Truncate word_a: all letters UNTIL and EXCLUDING the last vowel
+  # Check for vowels
+  word_a.reverse!
+  new_word_a = ""
+  word_a.each_char.with_index do |letter, index|
+    if is_vowel?(letter) == true
+      new_word_a = "#{word_a[(index + 1)..-1]}"
+      new_word_a = new_word_a.reverse!
+      break
+    else
+      new_word_a = word_a.reverse!
+    end
+  end
+
+  # Get second word
+  print "What is the second word to combine? "
+  word_b = gets.chomp.downcase
+  until word_b.length >= 2
+    puts "Word must be greater than 1 character! Try again: "
+    word_b = gets.chomp.downcase
+  end
+  
+  # Truncate word_b: all letters AFTER and INCLUDING first vowel
+  # Check for vowels
+  new_word_b = ""
+  word_b.each_char.with_index do |letter, index|
+    if is_vowel?(letter) == true
+      new_word_b = "#{word_b[(index + 1)..-1]}"
+    else
+      new_word_b = word_b
```

Comment:
You're close, but I think this logic isn't quite right. Here are a couple observations:
- Since the letters from the second word are supposed to include the vowel, you should use `index` instead of `index + 1` on line 46.
- Because you have this `else` clause, if the word doesn't end in a vowel, you will use the whole word. This is because the last iteration sets `new_word_b` to be the original word. Instead, after line 46 you should have a `break` statement.